### DISCUSSION TITLE
[Feature Request] Option to Show/Hide Powered by umami Footer message

### DISCUSSION THREAD

**Discussion by mariushosting** [umami-software/umami#1199] [Ideas]
An option to disable or change the message in the footer "Powered by umami" would be very useful and would help to customize the page on the Share URL Stats .

**Comment by foxytocin** [comment]
Actually I think you can realize such things with custom .css files.
On the other hand, the software is completely free and indicating that it's umami is a nice way of showing appreciation.

**Comment by mariushosting** [comment]
Most of the free projects like uptime kuma, homarr, etc have decided to introduce this option. It is not a question of appreciation or not. If we use it we appreciate it. Umami may grow even more when customization is allowed.
